### DISCUSSION TITLE
incomplete transfer of information from the intermediate environment to the production environment

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1965] [Q&A] [answered]
Could you tell me why the data is not fully displayed when transferring information from the staging environment to production (https://github.com/ukraine-sdg/data/pull/2182). Data from the "data" folder was displayed, but not from the "meta" folder. How to fix the error? Thank you!

**Comment by brockfanning** [comment, marked as answer]
The issue in this case is that the data repository production deployments are supposed to automatically trigger the site repository production deployments, but that is not happening for some reason. We may need to update the automation workflow in the site/data starter projects. I'll investigate and follow up here. In the meantime, the workaround is to manually perform a production release on the site repository after each data release. The steps for that are:

1. Go to `/compare/master...develop` on your site repository. For example: `https://github.com/ukraine-sdg/site/compare/master...develop`
2. Click "Create pull request"
3. When the "Merge pull request" button becomes green (it may take a few minutes) then click "Merge pull request" and then confirm.

A more detailed description of those steps are here: https://open-sdg.readthedocs.io/en/latest/hosting/github-pages-production/#performing-a-production-deployment-site-repository
